clang/test/Driver/fno-plt.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang -### -c --target=aarch64 -fno-plt -Werror %s 2>&1 | FileCheck %s
+// RUN: %clang -### -c --target=x86_64 -fno-plt -Werror %s 2>&1 | FileCheck %s
+// RUN: %clang -### -c --target=aarch64 -fno-plt -fplt -Werror %s 2>&1 | FileCheck %s --check-prefix=NO
```

Comment:
There's quite many negations involved in this check line, can you rename the `NO` prefix to e.g. `PLT` or something like that, to signify the case when `-fplt` is in effect? Then again, using `--check-prefix=PLT` for the `aarch64-windows` target feels a bit misleading as well. Perhaps if we'd rename `CHECK` into `NOPLT` and `NO` into `DEFAULT`?